ExampleMod/Common/Players/ExampleRegenPlayer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.Players
+{
+	public class ExampleRegenPlayer : ModPlayer
+	{
+		public override ModPlayer.Regen ModifyLifeRegen() {
+			var modifiers = ModPlayer.Regen.Create(); // Creates the struct with all values configured to do no modifications.
+			modifiers.deltaRate = 3f; // Increases the rate at which player reaches maximum natural life regeneration for every multiple of 300 up to 3600. Positives for buffs, negatives for debuffs.
+			modifiers.deltaRegen = 6f; // Increases the life per second recovered by typically 6 / 2 per second. Positives for buffs, negatives for debuffs.
```

Comment:
why is everything /2 here? Can we make it more sensible?

Reply:
I've improved the documentation after rewriting some things further. The /2 was what we originally had.
There is a fair few random scalars in vanilla, so I'm finding hard pressed not to comment that it has the scalars (although notably, the new documentation doesn't rely on actually knowing the scalars)